free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There are many people struggling with the same problem: how do we place things into groups based on an assortment of criteria. Much of this sort of problem solving is done by hand, but as the domain becomes larger, this becomes nearly impossible. I would like to create a language to help these people. Since this is an extremely broad scope, I would specifically focusing on creating a language which clinic organizers can use to place students on clinic teams based on what position they like, what projects they are interested in, who they want to work with or do not want to work with, and perhaps even who thy can easily meet with on a weekly basis. However, this should be extensible to more general group organisation for other activities as well. 
+
+We need better ways to describe a set of constraints and relationships to organize teams for things like clinic. Many students become very frusterated with clinic because they don't work well with the students on their team or don't like their project. If we could create a language that would make optimizing the team divisions easier, everyone might have a more positive clinic experience. 
 
```

Comment:
I'm not sure this is entirely true--I'm very happy with my team and I believe most of my friends are as well. But still, prioritizing good working relationships is important.